### ISSUE TITLE
Multiple h1s

### ISSUE THREAD

**Issue by oliverjam** [fac20/Week1-BDIM-DigitalAgency#51]
It looks like you've implemented the HTML5 document outline where each section starts a new "heading context" with a new `h1` inside each. Unfortunately no browser ever actually implemented this. Sections don't restart the heading hierarchy, so using multiple `h1`s will cause issues for assistive tech.

See the warning on MDN here: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines

tl;dr you can only have one `h1`, and then headings always have to go downwards in hierarchy.

**Comment by oliverjam** [comment]
Actually other than the multiple `h1`s your hierarchy is really good—you've just skipped over the `h2` tier. Just swapping the section `h1`s to `h2`s would probably be great
